src/screens/CredentialCatalogScreen/index.tsx

```diff
@@ -0,0 +1,222 @@
+import React, {FC, ReactElement} from 'react';
+import {RefreshControl, ListRenderItemInfo} from 'react-native';
+import {SwipeListView} from 'react-native-swipe-list-view';
+import {NativeStackScreenProps} from '@react-navigation/native-stack';
+import {IBasicCredentialLocaleBranding} from '@sphereon/ssi-sdk.data-store';
+import {backgroundColors, borderColors} from '@sphereon/ui-components.core';
+import {SSITextH2SemiBoldLightStyled as HeaderCaption} from '@sphereon/ui-components.ssi-react-native';
+import CredentialCardPreviewView from '../../components/views/CredentialCardPreviewView';
+import SearchField from '../../components/fields/SearchField';
+import FilterBar from '../../components/bars/FilterBar';
+import CredentialPreviewViewItem from '../../components/views/CredentialPreviewViewItem';
+import {OVERVIEW_INITIAL_NUMBER_TO_RENDER} from '../../@config/constants';
+import Localization from '../../localization/Localization';
+import {
+  CredentialCatalogScreenPreviewCredentialContainerStyled as PreviewCredentialContainer,
+  CredentialCatalogScreenRelevantCredentialContainerStyled as RelevantCredentialContainer,
+  CredentialCatalogScreenRelevantCredentialHeaderContainerStyled as RelevantCredentialHeaderContainer,
+  CredentialCatalogViewAllContainerStyled as ViewAllContainer,
+  CredentialCatalogViewAllTextStyled as ViewAllText,
+  SSIBasicContainerStyled as Container,
+  SSIRippleContainerStyled as ItemContainer,
+  CredentialCatalogDiscoverCredentialsContainerStyled as DiscoverCredentialsContainer,
+  CredentialCatalogScreenDiscoverCredentialsHeaderContainerStyled as DiscoverCredentialsHeaderContainer,
+  CredentialCatalogCredentialListContainerStyled as CredentialListContainer,
+  CredentialCatalogScreenPreviewCredentialContentContainerStyled as PreviewCredentialContentContainer,
+} from '../../styles/components';
+import {MainRoutesEnum, ScreenRoutesEnum, StackParamList} from '../../types';
+
+type Props = NativeStackScreenProps<StackParamList, ScreenRoutesEnum.CREDENTIAL_CATALOG>;
+
+type CredentialPreview = {
+  title: string;
+  issuer: string;
+  branding?: IBasicCredentialLocaleBranding;
+};
+
+const CredentialCatalogScreen: FC<Props> = (props: Props): ReactElement => {
+  const [refreshing, setRefreshing] = React.useState(false);
+
+  const credentials: Array<CredentialPreview> = [
+    {
+      title: 'Girocard',
+      issuer: 'EURO Kartensysteme GmbH',
+      branding: {
+        background: {
+          image: {
+            uri: 'https://i.ibb.co/NngL9SD/giro-Card-Background.png',
+          },
+        },
+        logo: {
+          uri: 'https://i.ibb.co/P928gkN/giro-Card-Logo.png',
```

Comment:
Could we download these and provide as assets. Don't want to stand in a venue with choppy internet connection and the screen loading all these images slowly. It is a mock/demo for the most part anyway